Pipfile

```diff
@@ -12,9 +12,9 @@ plexapi = "==4.6.1"
 python-dotenv = "==0.15.0"
 python-git-info = "==0.7.1"
 requests = ">=2.25.1"
-requests-cache = "==0.7.0.dev496"
+requests-cache = "==0.7.1"
 trakt = "==3.2.0"
-urllib3 = ">=1.26.5"
+urllib3 = ">=1.26.6"
```

Comment:
revert or fix commit/pull request title

Reply:
the dependancy chain is : requests-cache > requests > urllib3